src/utilities/licensePlate/index.test.ts

```diff
@@ -0,0 +1,52 @@
+// Tests created using the following links as reference:
+// https://help.returnpath.com/hc/en-us/articles/220560587-What-are-the-rules-for-email-address-syntax-
+// https://stackoverflow.com/questions/2049502/what-characters-are-allowed-in-an-email-address
```

Comment:
Should this be here?

Reply:
Should not, my bad. I made the PR fallowing the email as a template, sorry.